File: Assignment22/C/Program5.c
// PS - Accept string from user and display string in reverse order

#include <stdio.h>

void StringRev(char *str)
{
    int iLength = 0;

    while (*str != '\0')
    {
        iLength++;
        str++;
    }
    str--;

    while (iLength != 0)
    {
        printf("%c", *str);
        str--;
        iLength--;
    }
    
}

int main()
{
    char Arr[20];

    printf("Enter string \n");
    scanf("%[^'\n']s", Arr);

    StringRev(Arr);

    return 0;
}

File: Assignment34/Java/Program5.java
// PS - Write a java program which accept N numbers from user and display all such numbers which are multiplies of 11

import java.util.*;

class ArrayX
{
    public void MultipliesOf11(int Arr[])
    {
        for(int iCnt = 0; iCnt< Arr.length; iCnt++)
        {
            if(Arr[iCnt] % 11 == 0)
            {
                System.out.print(Arr[iCnt]+"\t");
            }
        }
    }
}

class Program5
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        int iSize = 0;
        System.out.println("Enter number of elements ");
        iSize = sobj.nextInt();

        int Arr[] = new int[iSize];

        System.out.println("Enter elements ");
        for(int iCnt = 0; iCnt < iSize; iCnt++)
        {
            Arr[iCnt] = sobj.nextInt();
        }

        ArrayX aobj = new ArrayX();
        aobj.MultipliesOf11(Arr);

    }
}